applications/DEMApplication/custom_constitutive/DEM_KDEM_with_damage_parallel_bond_CL.cpp

```diff
@@ -2,6 +2,21 @@
 #include <string>
 #include <iostream>
 #include <cmath>
+#include <cstdio>
+#ifdef _WIN32
+#include <direct.h>
+#define GetCurrentDir _getcwd
+#else
+#include <unistd.h>
+#define GetCurrentDir getcwd
+#endif
+
+std::string GetCurrentWorkingDir( void ) {
```

Comment:
all this part (including defines) is no longer necessary. At least I can't find where this function is called